app/models/activity.rb

```diff
@@ -82,6 +82,16 @@ class Activity < ApplicationRecord
                                    .confirmed.by_dates(period_begin, period_end)
                                    .order(occur_at: 'desc')
                                }
+  def accompaniable?
+    event != 'family_court' || accompaniments.count < 3
+  end
+
+  def volunteers_needed
```

Comment:
This message is really helpful to clarify for the user!  I kind of also wanted it when the cap was met.  I think instead of 'if accompaniable?' we could do "if event == 'family_court'"